18-08-2019:16:59:53,515 INFO [test_1.py:108] 退出成功
